modules/developer-cli-odo-ref-registry.adoc

```diff
@@ -0,0 +1,92 @@
+:_content-type: REFERENCE
+[id="odo-registry_{context}"]
+= odo registry
+
+
+`odo` uses the portable _devfile_ format to describe the components. `odo` can connect to various devfile registries, to download devfiles for different languages and frameworks.
+
+You can connect to publicly available devfile registries, or you can install your own _Secure Registry_.
```

Comment:
```suggestion
You can connect to publicly available devfile registries or you can install your own _Secure Registry_.
```

Reply:
I believe you should always place a comma before "or" when it begins an independent clause